Documentation/commands.md

```diff
@@ -209,8 +209,37 @@ FOO=bar
 EXAMPLE_OVERRIDE=over
 ```
 
+#### Logging
+
+To get the logs you need to get pod's machine name. You can use machinectl
+
+```
+$ machinectl
+MACHINE                                  CLASS     SERVICE
+rkt-f241c969-1710-445a-8129-d3a7ffdd9a60 container nspawn
+
+1 machines listed.
+```
+
+or `rkt list --full`
+
+```
+# rkt list --full
+UUID                                    ACI     STATE   NETWORKS
+f241c969-1710-445a-8129-d3a7ffdd9a60    busybox running
+```
+
+Pod's machine name will be the pod's UUID with a `rkt-` prefix.
+
+Then you can use systemd's journalctl with `_HOSTNAME` set to the machine name:
+
+```
+$ journalctl -m _HOSTNAME=rkt-f241c969-1710-445a-8129-d3a7ffdd9a60
```

Comment:
thanks for the info; this divergent behaviour feels to me like a bug in journalctl (if `systemctl -M`/`machinectl` work , and the host journal is still able to access the logs, seems like they should be referenceable directly..)